addon/session-stores/cookie.js

```diff
@@ -143,8 +166,7 @@ export default BaseStore.extend({
   },
 
   _read(name) {
-    let value = document.cookie.match(new RegExp(`${name}=([^;]+)`)) || [];
-    return decodeURIComponent(value[1] || '');
+    return this.get('_cookies').read(name) || '';
```

Comment:
This is throwing an error in FastBoot because recent versions of Ember don't like injected properties in objects without a container:

```
Error: Assertion Failed: Attempting to lookup an injected property on an object without a container, ensure that the object was instantiated via a container.
```